### ISSUE TITLE
BTCV benchmark dataset partition

### ISSUE THREAD

**Issue by cugwu** [ljwztc/CLIP-Driven-Universal-Model#7]
Hi, first of all, I appreciate your work. 
In your paper on Table 3, there is a benchmark on the BTCV validation test set. Is it possible to share the train/validation split you use? 
Or, did you use 5-fold cross-validation on the 30 available CT scans and you reported on the table the results on the best fold? 
Have you used five-fold cross-validation, reporting the average per organ validation accuracy across the five folds on the table?
Thanks in advance for the answer.

**Comment by ljwztc** [comment]
We use the 5-cross validation on 30 CTs and report the average results. The split is added to `dataset/BTCV_folds.json`.

**Comment by cugwu** [comment]
Perfect thanks for the answer
